src/build.ts

```diff
@@ -46,7 +46,14 @@ const go = async function (): Promise<void> {
       .join(", ")} for version: ${version}`
   );
   for (let i = 0; i < platarchs.length; i++) {
-    await fetchAndUnpack(platarchs[i]);
+    try {
+      await fetchAndUnpack(platarchs[i]);
+    } catch (err) {
+      // If the version of this package doesn't match a release there will not be anything to fetch,
+      // but we don't want the build to totally fail in case someone wants to publish a custom build
+      console.log("could not fetch:", platarchs[i]);
+      console.log(err);
+    }
```

Comment:
@joewagner it seems like we're leaving this in a kind of half baked state... If we're really going to decouple the version number of this `@tableland/validator` module from the version of `go-tableland`, there is no reason to still attempt to fetch the validator builds based on this module's version. Seems it would be better to just configure the desired go-validator version as a `const` at the top of this file? Or some other way?

Reply:
I just pushed the change to make the validator version a `const` at the top of this file. I was going to say, we can remove this change, but it seems like a good change to leave for the reason described in the comment.

Reply:
You've got a good point.  The reasoning I had for the "half baked" state was to automate keeping this package up to date with releases of go-tableland, but still be able to add custom builds. Mostly to enable adding M1 builds as needed until github actions support them.  
If we configure the go-tableland version as a `const` at the top of the file it would mean that one of us has to manually intervene for each release. This is ok with me, but it will take a little more input from one of us.  Maybe that's better since someone will have to do the M1 build anyway?